src/main/scala/com/peoplepattern/text/Implicits.scala

```diff
@@ -1,23 +1,72 @@
 package com.peoplepattern.text
 
+import java.net.{ URL, MalformedURLException }
+
 object Implicits {
 
+  /** Helpers to provide access to LangBundle and StringUtil stuff on strings */
   implicit class StringWithAnalysis(str: String) {
 
+    /** The detected language of the string, if predicted */
     lazy val lang: Option[String] = LanguageIdentifier.classify(str).map(_._1)
 
+    /** The language bundle for the string based on it's predicted language */
     lazy val bundle: LangBundle = LangBundle.bundleForLang(lang)
 
+    /** The tokens of the string using language specific tokenization */
     lazy val tokens: Vector[String] = bundle.tokens(str)
 
+    /** The terms of the string using language specific tokens & stopwords */
     lazy val terms: Set[String] = bundle.terms(tokens)
 
+    /** The terms + hashtags + @-mentions of the string */
     lazy val termsPlus: Set[String] = bundle.termsPlus(tokens)
```

Comment:
Let me think about this. It's outside the scope of this PR but something along those lines may be worth doing separately.